script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -570,14 +582,14 @@ However, it requires a millennium-class basis in order to command a Hemonomic Pr
 [sha:4fde2c2628b5601d767c29c05646c69e0838cc83]{
 -- Page 612, Offset 18115.
 -- 「そうなんだけど……今にして思うと、アイツ、ちゃんと耐えきっていたみたい。
-"Sure, but... now that I think about it, that guy looked like he was able to withstand it pretty well.
+"Sure, but... now that I think back on it, that guy looked like he was able to withstand it pretty well.
 }
 [sha:601c575a014f7a176e961bc5b894fc7a5827c162]{
 -- Page 612, Offset 18116.
 -- ……死徒としてはⅥ階梯クラスだったのに、なんであそこまで自我が残っていたのかしら……」
 // Vlov is _technically_ rank IX, but actually only really as capable as VI
-He was only in the same class as a Rank VI Dead Apostle,
- I'm not sure how he was able to retain as much of his ego as he did..."
+He was only at a Rank VI as far as Dead Apostles go,
```

Comment:
He was only a Rank VI

Reply:
Yeah "He was only Rank VI" seems like the best rendition